src/main/java/at/hannibal2/skyhanni/features/inventory/chocolatefactory/hitman/HitmanApi.kt

```diff
@@ -24,6 +30,18 @@ object HitmanApi {
     private val orderOrdinalMap = resettingEntries.mapIndexed { index, hoppityEggType ->
         hoppityEggType to resettingEntries[(index + 1) % resettingEntries.size]
     }.toMap()
+    private val storage get() = ProfileStorageData.profileSpecific
+
+    @HandleEvent
+    fun onInventoryFullyOpened(event: InventoryFullyOpenedEvent) {
+        if (event.inventoryName != "Rabbit Hitman") return
+        val summaryItem = event.inventoryItems[4].takeIf { it?.displayName == "§cRabbit Hitman" } ?: return
+        val availableEggs = hitmanAvailableEggsPattern.firstMatcher(summaryItem.getLore()) {
+            group("amount").toInt()
+        } ?: return
+        val storage = storage ?: return
```

Comment:
move storage to first line of this function. there i no need to have the code above run at all if this crucial (and easy to check) precondition is not met